services/catalog/src/simcore_service_catalog/api/rest/_services.py

```diff
@@ -189,7 +190,9 @@ async def cached_registry_services() -> dict[str, Any]:
 )
 async def get_service(
     user_id: int,
-    service: Annotated[ServiceGet, Depends(get_service_from_registry)],
+    service_in_manifest: Annotated[
```

Comment:
Q: just the naming is odd to me, this is the `service_manifest` right? I don't fully understand what `service_in_manifest` hints at

Reply:
This manifest is not the docker registry manifest. For that reason I used `in` as in the `catalog's manifest'